zebra-test/src/mock_service.rs

```diff
@@ -959,6 +959,7 @@ impl<Request, Response, Error> ResponseSender<Request, Response, Error> {
     }
 }
 
+#[allow(dead_code)]
```

Comment:
###### Optional Suggestion

It's better to put attributes below comments: https://github.com/rust-lang/style-team/issues/72.